infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -165,6 +165,11 @@ public static void addLegacyTypesToSerializer(CorfuStore corfuStore) {
                 .addTypeToClassMap(LogReplicationMetadata.ReplicationEventKey.getDefaultInstance());
     }
 
+    public static boolean isMigrationComplete(TxnContext txnContext) {
+        Table<?,?,?> table = txnContext.getTable(LogReplicationUtils.REPLICATION_STATUS_TABLE_NAME);
```

Comment:
hi Shama, when the node first comes up - this table would not even be populate in which case you would get a `NoSuchElementException` right? Won't you need to surround this with `try-catch` blocks?

Reply:
this logic is no longer there with the recent push